bin/node/runtime/src/lib.rs

```diff
@@ -640,17 +644,19 @@ frame_election_provider_support::generate_solution_type!(
 		VoterIndex = u32,
 		TargetIndex = u16,
 		Accuracy = sp_runtime::PerU16,
-		MaxVoters = MaxElectingVoters,
+		MaxVoters = MaxNominations,
 	>(16)
```

Comment:
Curious if this `16` could now be replaced with a const `MAX_QUOTA_NOMINATIONS`.  Might not be possible due to macro limitations.